test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -6,17 +6,21 @@
 from parameterized import parameterized
 
 from torchaudio_unittest import common_utils
+from torchaudio_unittest.common_utils import TempDirMixin, TestBaseMixin
 from torchaudio_unittest.common_utils import (
     skipIfRocm,
 )
 
 
-class Functional(common_utils.TestBaseMixin):
+class Functional(TempDirMixin, TestBaseMixin):
     """Implements test for `functinoal` modul that are performed for different devices"""
     def _assert_consistency(self, func, tensor, shape_only=False):
         tensor = tensor.to(device=self.device, dtype=self.dtype)
 
-        ts_func = torch.jit.script(func)
```

Comment:
we shouldn't remove this and continue checking `scripted_fn(input) == fn(input)`

Reply:
I agree with the above. To clarify, `torch.jit.load` returns a scripted module with a `forward` method equivalent to the function that was serialized. The caller does not notice the difference because `__call__` for the module calls `forward`.

Reply:
Hmm, I have a mixed feeling about introducing another helper function.

First, this method **IS** the helper function used by the actual tests.
Secondly, my view is that if it's something that needs explanation with comment, keeping it along side of tests make it easier to follow the logic of tests.

Reply:
At the moment, you duplicate the same code logic at different places, which I think strongly suggests that we should just have one helper function that can be used in all the tests.